easybuild/framework/easyblock.py

```diff
@@ -2952,6 +2952,12 @@ def apply_post_install_patches(self, patches=None):
             # To allow postinstallpatches for Bundle, and derived, easyblocks we directly call EasyBlock.patch_step
             EasyBlock.patch_step(self, beginpath=self.installdir, patches=patches)
 
+    def log_post_install_messages(self):
```

Comment:
`log_` is a bit of a misnomer, may use `print_post_install_messages`?

Please add a docstring to this new function as well.